client/my-sites/customer-home/main.jsx

```diff
@@ -178,21 +178,20 @@ const Home = ( {
 				<JetpackConnectionHealthBanner siteId={ siteId } />
 			) }
 			{ header }
-			{ isLoading ? (
-				<div className="customer-home__loading-placeholder"></div>
-			) : (
+			{ isLoading && <div className="customer-home__loading-placeholder"></div> }
+			{ ! isLoading && ! homeLayoutError ? (
```

Comment:
```suggestion
			{ ! isLoading && layout && ! homeLayoutError ? (
```

Should we check for `layout` here, instead of making it optional below? Is there some scenario where `! homeLayoutError` but `layout` is still empty?

Reply:
The problem is not if it's empty, but if it's `undefined` or `null`. However, as I don't trust js, I totally agree with your suggestion!